pkg/spireentry/entries_test.go

```diff
@@ -45,7 +46,7 @@ func TestRenderPodEntry(t *testing.T) {
 	td, err := spiffeid.TrustDomainFromString(trustDomain)
 	require.NoError(t, err)
 
-	entry, err := renderPodEntry(parsedSpec, node, pod, td, clusterName)
+	entry, err := renderPodEntry(parsedSpec, node, pod, td, clusterName, clusterDomain)
```

Comment:
Can we add (or embellish an existing) test to make sure the clusterDomain is available to the template?

Reply:
Done: https://github.com/spiffe/spire-controller-manager/pull/90/commits/fbab85acb0a16d7061dd8cf25e604bde5ee2acf9